content/blog/Caldera: Pimp my TTPs.md

```diff
@@ -57,94 +57,84 @@ The platform offers comprehensive documentation covering installation, configura
 
 **Here we have the initial launch of the Caldera server on our attacking server:**
 
-![image](https://github.com/user-attachments/assets/b24a841f-5581-429c-8121-a1ecf23a14bc)
-
-
-**After connecting to the Caldera server on the login page here is the main login page:**
-
-![image](https://github.com/user-attachments/assets/8141089f-c788-47ce-9f0f-590c727e35c6)
+![Image3](https://github.com/user-attachments/assets/ff243ae9-92e0-4e26-82fe-94a26ca07e9b)
```

Comment:
I'm not sure these are kept indefinetely. Can you just put them in the repository itself please, like the other blog posts?
`/img/...`